public/script.js

```diff
@@ -40,28 +38,55 @@ function getSnippet() {
             } else if (data === "") {
                 alert('Unknown  error has occurred.')
             } else {
-                // sending data and speaking .
+                paragraph.innerHTML = data;
                 speech.text = data;
-                window.speechSynthesis.speak(speech)
-                paragraph.innerHTML = data
+
+                if (clicked) {
+                    window.speechSynthesis.speak(speech);
+                } else {}
+
                 setSucces(inputs)
             }
         })
     )}
 }
 
+//read button for readSnippet
+const repeat = document.getElementById("repeat")
+repeat.addEventListener('click',readSnippet )
+
+function readSnippet(){
+    window.speechSynthesis.speak(speech); 
+}
+
+// mute button for read snippet
+let clicked = false;
+const p = document.getElementById("output");
+const mute = document.getElementById("on");
+document.getElementById("on").onclick = toggle = () => {
+    if(clicked){
+        clicked = false;
```

Comment:
What is "clicked"? Give a better name for this variable.
"speechMuted" or something